### PR TITLE
Added actionType field to Event

### PR DESCRIPTION
**Pull request by kriscfoster**
**_api/v1/swagger.yaml_**
- Updated EventResponse
- Updated prams for /events routes

**_db/model/event.js_**
- Added actionType field

**_migrations/20190204134143-event-add-type.js_**
- Migration for actionType field

Tests

### CODE REVIEW

File: api/v1/swagger.yaml [hunk 1]
```diff
@@ -3829,6 +3829,9 @@ paths:
               context:
                 type: object
                 description: Any object, typically JSON, used to describe to some infomration that needed to be logged.
+              type:
```
Comment (iamigo) [inline api/v1/swagger.yaml]: what about adding type to the list of attributes you can filter from query params, e.g. ...&type=foo

File: db/model/event.js [hunk 2]
```diff
@@ -33,6 +34,11 @@ module.exports = function event(seq, dataTypes) {
       allowNull: false,
       comment: 'This is a readable event logline',
     },
+    type: {
+      type: dataTypes.STRING(constants.fieldlen.normalName),
```
Comment (iamigo) [inline db/model/event.js]: free text here is recipe for chaos! what are the values you actually expect to see here? would it be possible to make it an enum with "Other" as one of the options? then add options as needed down the line? it's not going to be infinite! i'm sure you can come up with a half dozen values or fewer

Comment (kriscfoster) [reply db/model/event.js]: In this case, I think free text is actually the best option for us. The "type" is defined by the bot developer. e.g. We will have a type values of: ["sendEmailTCE", "uploadFileToCase", "pageTeam", "createSlackChannel", ...] .. If a new bot developer wants to create events of their own unique type, they should be able to do that without needing to make changes to the platform

File: api/v1/swagger.yaml (hunk 1 above)
Comment (kriscfoster) [reply api/v1/swagger.yaml]: Thanks Ian, good catch, done

File: db/model/event.js (hunk 2 above)
Comment (iamigo) [reply db/model/event.js]: I agree with you, but I also still think it's a recipe for chaos... oh well

File: api/v1/swagger.yaml [hunk 3]
```diff
@@ -3889,6 +3898,9 @@ paths:
                 context:
                   type: object
                   description: Any object, typically JSON, used to describe to some infomration that needed to be logged.
```
Comment (BrandonMurnane) [inline api/v1/swagger.yaml]: also here information

**Pull request merged**